script.js

```diff
@@ -1,4 +1,228 @@
+    }
+
+  }
+  //Player chose to play normal mode
+  else if (currentGameMode=='now playing' && chosenGameMode == 'normal'){
+    var compChoice = compRandomChoice()
+    var playerChoice = playerInput
+    //check if input is valid
+    if (checkValidInput(playerChoice) == true){
+      var myOutputValue= normalGameMode(playerChoice, compChoice, formatToObject(playerChoice), formatToObject(compChoice))
+    }
+    else{
+      var myOutputValue = `You chose ${playerChoice}, that is invalid input! Please type: 'scissors', 'paper' or 'stone'`
+    }
+  }
+  //Player chose to play reverse
+   else if (currentGameMode=='now playing' && chosenGameMode == 'reverse'){
+    var compChoice = compRandomChoice()
+    var playerChoice = playerInput
+    //check if input is valid
+    if (checkValidInput(playerChoice) == true){
+      var myOutputValue= reverseGameMode(playerChoice, compChoice, formatToObject(playerChoice), formatToObject(compChoice))
+    }
+    else{
+      var myOutputValue = `You chose ${playerChoice}, that is invalid input! Please type: 'scissors', 'paper' or 'stone'`
+    }
+  }
+  //player chose to play korean
+    else if (currentGameMode=='now playing' && chosenGameMode == 'korean'){
+    var compChoice = compRandomChoice()
+    var playerChoice = playerInput
+    //check if input is valid
+      if (checkValidInput(playerChoice) == true){
+        var myOutputValue= koreanGameMode(playerChoice, compChoice, formatToObject(playerChoice), formatToObject(compChoice))
+    }
+      else{
+        var myOutputValue = `You chose ${playerChoice}, that is invalid input! Please type: 'scissors', 'paper' or 'stone'`
+    }
+  }
+  //player chose to play computer vs computer
+    else if (currentGameMode=='now playing' && chosenGameMode == 'computer'){
+    var compChoice = compRandomChoice()
+    var playerChoice = compRandomChoice()
+    //check if input is valid
+      if (checkValidInput(playerChoice) == true){
+        var myOutputValue= computerGameMode(playerChoice, compChoice, formatToObject(playerChoice), formatToObject(compChoice))
+    }
+      else{
+        var myOutputValue = `You chose ${playerChoice}, that is invalid input! Please type: 'scissors', 'paper' or 'stone'`
+    }
+  }
   return myOutputValue;
 };
+
+//Function for Normal Game Mode
+var normalGameMode = function (playerChoice, compChoice, playerChoiceFormatted, compChoiceFormatted){
+  totalGamesPlayed = totalGamesPlayed+1
+//   var gameStatus = scoreBoard(userName, playerScore, compScore, totalGamesPlayed)
+if (playerChoice == compChoice){
+  totalDraw = totalDraw+1
+  var gameStatus = scoreBoard(userName, playerScore, compScore, totalGamesPlayed,totalDraw)
+  var outputMessage = `It's a draw! 😃 <br> You chose ${playerChoiceFormatted} <br> The Computer chose ${compChoiceFormatted}<br><br> ${gameStatus}`
+  return outputMessage
+  }
+
```

Comment:
Nice use of the or logical operator! 
 you could separate game logic into smaller chunks of functions. For example, you could have one function specifically to check for who wins in each round and and a gamePlay function to execute game logic. Well done though with your code!